push/mv.go

```diff
@@ -207,6 +208,21 @@ func (m *MaterializedView) Fill(shardIDs []uint64, interruptor *interruptor.Inte
 		return errors.WithStack(err)
 	}
 
+	// Start any sources that wait to ingest on first MV
+	for _, tableExec := range tes {
+		if tableExec.IngestOnFirstMV() {
```

Comment:
This information is already available on the SourceInfo. You can look that up in the meta using the table ID

Reply:
Gotcha. Removed from table_exec. Now I'm looking up the table and if it's a source, getting that flag value from SourceInfo as suggested 🙏